src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bar__container">
+      <form
+        class="search-bar search-bar_top"
+        data-qa="big"
+      >
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__input search-bar__input_top"
+        />
```

Comment:
Consider adding a `<label>` element for the input field to improve accessibility. This helps screen readers to identify the purpose of the input field.